libraries/collide2/CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2025 Benjamen Meyer and other Vega Strike contributors.
+#
+# https://github.com/vegastrike/Vega-Strike-Engine-Source
+#
+# This file is part of Vega Strike.
+#
+# Vega Strike is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Vega Strike is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
+
+MESSAGE("PROCESSING DIRECTORY: ${CMAKE_CURRENT_SOURCE_DIR}")
+
+LIST(APPEND
+    OPCOLLIDE_INCLUDES
+    # collide2 headers
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    # VS engine headers
+    #${CMAKE_SOURCE_DIR}/engine
+    ${CMAKE_SOURCE_DIR}/engine/src
+    ${CMAKE_SOURCE_DIR}/engine/src/cmd
+    ${CMAKE_SOURCE_DIR}/engine/src/components
+    ${CMAKE_SOURCE_DIR}/engine/src/damage
+    ${CMAKE_SOURCE_DIR}/engine/src/resource
+    ${CMAKE_SOURCE_DIR}/engine/src/python/base_computer
+    ${CMAKE_SOURCE_DIR}/engine/src/python/config
+    # CMake Artifacts
+    ${Vega_Strike_BINARY_DIR}
+    ${Vega_Strike_BINARY_DIR}/engine
+    ${Vega_Strike_BINARY_DIR}/src
+)
+MESSAGE("OpCollide Includes: ${OPCOLLIDE_INCLUDES}")
+
+ADD_LIBRARY(vegastrike-OPcollide STATIC
```

Comment:
Looks like the `STATIC` clause is new here. That might be part of the problem. We link many, if not all, of the libraries dynamically on Windows. Possibly on macOS as well.